course/assets/js/course_info_toggle.ts

```diff
@@ -29,30 +31,37 @@ export const initDesktopCourseInfoToggle = () => {
 }
 
 function calculateTableResponsiveness() {
-  const table = document
-    .getElementById("main-content")
-    .getElementsByTagName("table")[0]
-  const mainContentWidth = document.getElementById("main-content").clientWidth
-  if (table.clientWidth > mainContentWidth) {
```

Comment:
Two issues here:
1. This was running whether or not the table existed. So every non-table page was throwing an error.
2. Also, we were only being responsive for the first table on each page.

Fix is:
- be responsive for all tables on the page (all might be zero).

We should use typescript. Typescript would have caught this error. (At least error 1)